### ISSUE TITLE
feature request - define multiple timings?

### ISSUE THREAD

**Issue by dheise-rheinenergie-trading-com** [jhuckaby/Cronicle#371]
## Summary
It would be very useful if it would be possible to define multiple timings for an event. If I want to run an event at 08:30 and 11:00 o'clock, I have to set the timing to :00, :30, 08: and 11:. So it runs 08:00, 08:30, 11:00 and 11:30. 2 times more than I really need.

## Steps to reproduce the problem

See summary.

## Your Setup
Cronicle 0.8.53 running on 3 servers.

### Operating system and version?
Ubuntu 18.04

### Node.js version?
v.8.10

### Cronicle software version?
0.8.53

### Are you using a multi-server setup, or just a single server?
multi-server

### Are you using the filesystem as back-end storage, or S3/Couchbase?
file system

### Can you reproduce the crash consistently?
no crash

## Log Excerpts
no logs

**Comment by mikeTWC1984** [comment]
There might be somewhat nice and simple solution for that. Basically extend regular schedule with custom expression.
The expression could look like "08:30, 11:30", which is human readable/writable and fairly simple to parse. I'm experimenting with it on my custom fork, I'll share my finding later

**Comment by dheise-rheinenergie-trading-com** [comment]
something like this whould be great! adding/removing multiple timings.
![image](https://user-images.githubusercontent.com/73586581/106256700-e7f27e00-621b-11eb-8f62-ef84e2427645.png)

**Comment by mikeTWC1984** [comment]
I've been experimenting with this for a while. That extra schedule option could look like this:
![image](https://user-images.githubusercontent.com/31977106/111893328-61b31680-89d8-11eb-812b-e13d06a14fd2.png)
If you are using uneven schedules frequently you can [try it out](https://github.com/cronicle-edge/cronicle-edge). Although to cover simple cases like described in the issue you  can create an extra event that would do nothing but chaining your target event (just on different schedule)

**Comment by dheise-rheinenergie-trading-com** [comment]
Hi @mikeTWC1984,
I looked into your fork which seems to have a lot of cool new features.

Is it fully compatible with the original Cronicle, so that I easily can switch my servers to your version?

Btw: I solved my problem with a "one-liner-script" which is able to skip the execution of my event to unwanted times.

Daniel

**Comment by mikeTWC1984** [comment]
Yes, it's the same thing as Cronicle 0.8.56, just with few extra api/ui elements and some minimal refactoring.  You can use output of ```storage-cli.js export``` to import jobs from classic Cronicle. You can use UI for "safe" import (it would ignore server and plugin info). It also has a demo "export file" under sample_conf/backup.

**Comment by dheise-rheinenergie-trading-com** [comment]
Is it also possible to simply copy the config and data folder from the original installation to the new edge installation?

**Comment by mikeTWC1984** [comment]
I think you can copy data/logs to import run history/activity log, and data/jobs to import actual job logs (whatever you see on Job Details page). As I mentioned you can import metadata via UI, but you need to set up servers and custom plugins separately (manually). Anyway, you probably can start with importing metadata only and play around. Hopefully we'll see v2.0 of Cronicle in the near future, which should be main target for switching.
